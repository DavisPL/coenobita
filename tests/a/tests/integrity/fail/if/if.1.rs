#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::integrity({root}{root})]
    let mut x = 5;

    if b::boolean() {
        x = 6;
    } else {
        x = 7;
    }
}