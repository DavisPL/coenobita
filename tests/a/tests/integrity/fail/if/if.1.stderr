error: expected {root}{root}, found {root}{b,root}
 --> tests/integrity/fail/if/if.1.rs:9:13
  |
9 |         x = 6;
  |             ^
